services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/diagnostic_reports/question_report.jsx

```diff
@@ -10,13 +10,14 @@ export default class QuestionReport extends React.Component {
     };
   }
 
-  columnDefinitions = () => {
+  columnDefinitions = (activityClassification) => {
     return [
       {
         name: 'Score',
         field: 'score',
         sortByField: 'score',
         customCell: function(row) {
+          if (activityClassification === 'comprehension') return 'N/A'
```

Comment:
Inclined to say it probably makes sense to put `'comprehension'` in a constant in a shared file here too (maybe in the `progress_reports` subfolder?) since it is getting used for business logic throughout these files.